x/gamm/pool-models/stableswap/pool_test.go

```diff
@@ -1088,6 +1089,75 @@ func TestValidatePoolLiquidity(t *testing.T) {
 	}
 }
 
+func TestSetScalingFactors(t *testing.T) {
+	pk := ed25519.GenPrivKey().PubKey()
+	addr := sdk.AccAddress(pk.Address())
+
+	failPk := ed25519.GenPrivKey().PubKey()
+	failAddr := sdk.AccAddress(failPk.Address())
```

Comment:
Can we instead use one of `suite.TestAccs` addresses?

Reply:
This test, it using `testing.T` instead of `suite` so I think its take a lot of code to setup `suite` for it